main/Kconfig.projbuild

```diff
@@ -100,22 +91,18 @@ menu "User Configuration"
 
     config EMONCMS_HOST
         string "Hostname of Emoncms server"
-        depends on EMONCMS
         default "emoncms.org"
 
     config EMONCMS_PORT
         string "Port of Emoncms server"
-        depends on EMONCMS
         default "80"
 
     config EMONCMS_URL
         string "URL of Emoncms post API endpoint"
-        depends on EMONCMS
         default "/emoncms/input/post"
 
     config EMONCMS_APIKEY
         string "API key for Emoncms access"
-        depends on EMONCMS
         default ""
 
     config EMONCMS_NODE_SERIAL
```

Comment:
`depends on EMONCMS` has to be removed here as well, otherwise I get a compile error.